workers/Dockerfile.tests

```diff
@@ -0,0 +1,3 @@
+FROM dr_worker
```

Comment:
Ah.. just saw your revert.

So were you trying to take advantage of Docker's layering to do less rebuilding? Sounds good in theory, but not if it doesn't work, obviously. Maybe Docker needs a dependency tree feature so developers can link together related repositories and automatically rebuild upstream as necessary.